conductor/__beta__/job/maya.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from . import job
+
+LOG = logging.getLogger(__name__)
+
+class MayaRenderJob(job.Job):
+    
+        def __init__(self, scene_path=None, project_path=None, *args , **kwargs):
+            
+            super(MayaRenderJob, self).__init__(*args, **kwargs)
+            
+            self.cmd = "Render"
+            self.render_layer = "defaultRenderLayer"
+            self.scene_path = scene_path
+            self.project_path = project_path
+            
+            self.software_packages_ids = ["bcfe5df6e2361d77ca7d7b9da76e351b", "936dac0a489071942be623da35dd71fb"]
+            self.upload_paths.append(scene_path)
+            
+        def _get_environment(self):
+            return {"MAYA_PLUG_IN_PATH": "/opt/solidangle/arnold-maya/3/arnold-maya-maya2018-3.2.1-1./plug-ins", "MAYA_LOCATION": "/opt/autodesk/maya-io/2018/maya-io2018.SP6", "MAYA_LICENSE": "unlimited", "PYTHONPATH": "/opt/autodesk/maya-io/2018/maya-io2018.SP6/Conductor:/opt/solidangle/arnold-maya/3/arnold-maya-maya2018-3.2.1-1./scripts", "MAYA_DISABLE_CIP": "1", "MAYA_SCRIPT_PATH": "/opt/solidangle/arnold-maya/3/arnold-maya-maya2018-3.2.1-1./scripts", "solidangle_LICENSE": "4101@docker_host", "MAYA_RENDER_DESC_PATH": "/opt/solidangle/arnold-maya/3/arnold-maya-maya2018-3.2.1-1.", "PATH": "/opt/autodesk/maya-io/2018/maya-io2018.SP6/bin:/opt/solidangle/arnold-maya/3/arnold-maya-maya2018-3.2.1-1./bin", "ARNOLD_PLUGIN_PATH": "/opt/solidangle/arnold-maya/3/arnold-maya-maya2018-3.2.1-1./plug-ins", "LD_LIBRARY_PATH": "/opt/autodesk/maya-io/2018/maya-io2018.SP6/lib:/opt/autodesk/maya-io/2018/maya-io2018.SP6/plug-ins/xgen/lib:/opt/autodesk/maya-io/2018/maya-io2018.SP6/plug-ins/bifrost/lib"}
+        
+        def _get_task_data(self):
+            
+            {"frames": "1-2", "command": "Render  -s 1 -e 2 -b 1 -rl defaultRenderLayer -rd /tmp/render_output/ -proj \"C:/Users/jlehrman/Documents/maya/projects/conductor/\" \"/Users/jlehrman/Documents/maya/projects/conductor/simple_shapes.ma\""},

```

Comment:
?

Reply:
oops.. That was there for convenience - I'll fix it.